spec/abstract-operations.html

```diff
@@ -744,4 +744,17 @@ <h1>AddValueToTupleSequenceList ( _sequence_, _value_ )</h1>
       </emu-alg>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-isconcatspreadable" aoid="IsConcatSpreadable">
+      <h1>Runtime Semantics: IsConcatSpreadable ( _O_ )</h1>
+      <p>The abstract operation IsConcatSpreadable takes argument _O_. It performs the following steps when called:</p>
+      <emu-alg>
+        1. If Type(_O_) is not Object, return *false*.
```

Comment:
Oops, we already returned false up here! "is not Object or Tuple", probably this should say, with a GetV below.